src/utils/messages.js

```diff
@@ -1,20 +1,28 @@
-const generateMessage = (username, text) => {
-    return {
-        username,
-        text,
-        createdAt: new Date().getTime(),
-    }
-}
+const User = require('../entities/user')
 
-const generateLocationMessage = (username, url) => {
-    return {
-        username,
-        url,
-        createdAt: new Date().getTime(),
-    }
-}
+const admin = new User({ username: 'admin' })
+
+const makeMessageGenerator =
+    (author, process = (p) => p, payloadKey = 'text') =>
```

Comment:
Could you please clarify if this is a function, how are there no curly brackets after the "arrow"?

Reply:
Because it is kinda "one-line" arrow function,  https://javascript.info/arrow-functions-basics
It has only one logical line which is returned